framework/src/main/java/org/checkerframework/framework/util/TypeArgumentMapper.java

```diff
@@ -265,7 +265,7 @@ private static List<TypeRecord> depthFirstSearchForSupertype(
     }
 
     /**
-     * Computes one level for depthFirstSearchForSupertype then recurses.
+     * Computes one level for depthFirstSearchForSupertype then recurse.
```

Comment:
I think the original was correct and consistent with the `Computes`.
You could reformulate the whole sentence to make it clearer.